src/libraries/System.Data.Odbc/src/System/Data/Odbc/OdbcConnectionHandle.cs

```diff
@@ -57,9 +57,7 @@ private ODBC32.RetCode AutoCommitOff()
 
             Debug.Assert(HandleState.Connected <= _handleState, "AutoCommitOff while in wrong state?");
 
-            // Avoid runtime injected errors in the following block.
             // must call SQLSetConnectAttrW and set _handleState
-            RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
This project targets netstandard + netframework.
(This comment applies file-wide.)

Reply:
The file is in an ItemGroup that only targets NetCoreApp.